lms/djangoapps/support/static/support/js/views/enrollment_modal.js

```diff
@@ -55,6 +55,18 @@
                         // Success callback
                         _.bind(function() {
                             this.hide();
+                            /*
+                            * `/certificates/regenerate` makes the certificate status to unavailable (if it exists)
```

Comment:
Seems like a nice addition. 

One question in addition to this would be that if we get some error message while regenerating the certificate like changing from honor to audit, we may get  `Audit mode does not have certificate, current available modes are ['honor', 'verified']`, or something like that.  So should we show that error message and if yes, then where.